### DISCUSSION TITLE
再漫画部分漫画无法查看

### DISCUSSION THREAD

**Discussion by TkzcM** [xiaoyaocz/flutter_dmzj#227] [General]
已登录账号，网页和官方app可正常查看
![Screenshot_20240914-191534_ZAI-X](https://github.com/user-attachments/assets/8a5de7dd-2ed5-4978-8b4c-d35148964bde)

无法查看的几个漫画id：72043 74522 68755

**Comment by zhongfly** [comment]
具体什么名称啊？

update:用dmzj找到了，这些漫画在app里是仅登录后可见。应该是别的帖子说到的登录无效的原因
